### DISCUSSION TITLE
how to use my .obj coil file? or how to get .ccd file by utilizing .obj file?

### DISCUSSION THREAD

**Discussion by windyjunfeng** [simnibs/simnibs#241] [8. Simulation workflows]
Now I only have .obj format coil file. I find there isn't moment information. It only includes the vertices data and face data. However, I find the .ccd file provided by you includes the position and moment of dipoles. Does it means .obj file lack some information? How can I use my .obj file? Or how can I convert it to the .ccd file? Thank you very much.
![image](https://github.com/simnibs/simnibs/assets/58356387/2159b652-0bef-4a5c-9044-667797f059da)
![image](https://github.com/simnibs/simnibs/assets/58356387/c67b0a73-d710-46f3-869b-304cc1e1e703)

**Comment by TorgeW** [comment]
With our newest pre release we added a new coil file format. 
This includes the option to use a wire path to construct a coil.
You can also add a casing to your coil.
Examples can be found here:
https://github.com/simnibs/simnibs/tree/master/simnibs/examples/coils

In the case of your obj file, if its just the coil casing you are indeed missing information.
The important information you need would be the wire path of the coil

**Reply by windyjunfeng** [reply]
OK, I see. Thank you very much. My .obj file only includes surface data and lacks the specific wire information. Maybe I can compute the optimal position of the coil according to the .ccd file and then utilize the transformation matrix between the .ccd file and .obj file to get the real position. In this way, maybe it can apply to the neural navigation system?

**Reply by Kstoffer** [reply]
I'm unsure how the coordinate system is defined for you obj file, but yes if you know the relative position. then yes. The zero point for the ccd file is at the center surface of the coil (the hotspot of the coil, with z pointing into the head and y to the opposite of the handle).

**Reply by windyjunfeng** [reply]
OK, thank you very much. I made it~
